packages/aws-cdk-lib/aws-cloudwatch-actions/test/lambda.test.ts

```diff
@@ -111,3 +112,36 @@ def handler(event, context):
   });
 });
 
+test('can create multiple alarms for the same lambda if feature flag is set', () => {
```

Comment:
nit: Can we add another unit test that has feature flag set to false and create multiple alarms for the same lambda? The test should except to throw an exception to guarantee backward compatibility.